gokart/build.py

```diff
@@ -9,16 +9,19 @@
 from gokart.utils import check_config, read_environ
 
 
-class HideLogger:
-    def __init__(self, verbose: bool):
-        self.verbose = verbose
+class LoggerConfig:
+    def __init__(self, level: Optional[int]):
```

Comment:
@hirosassa How about     `def __init__(self, level: int = logging.CRITICAL): ` instead of Optional ?

Reply:
Are there any special reason to allow None instead of CRITICAL?

Reply:
There's no specific reason to allow that. Fixed in  60f7a65.